packages/vector_graphics/lib/src/listener.dart

```diff
@@ -48,6 +51,10 @@ class FlutterVectorGraphicsListener extends VectorGraphicsCodecListener {
       0, 0, 0, 0, 0,
       0.2126, 0.7152, 0.0722, 0, 0,
     ]); //convert to grayscale (https://www.w3.org/Graphics/Color/sRGB) and use them as transparency
+  static const ui.ParagraphConstraints _infiniteParagraphConstraints =
+      ui.ParagraphConstraints(
+    width: double.infinity,
+  );
```

Comment:
If this is only used in one place we might as well inline it.

Reply:
fixed